test/common/http/http1/codec_impl_test.cc

```diff
@@ -1,3 +1,5 @@
+#include <http_parser.h>
```

Comment:
See other comment on how we expose this and set it. I think we need to rethink this.

Reply:
Can this include be removed now?

Reply:
good catch!